pkg/util/proxyconfig/merge.go

```diff
@@ -23,6 +23,8 @@ func MergeUserSystemNoProxy(proxy *configv1.Proxy, infra *configv1.Infrastructur
 	set := sets.NewString(
 		"127.0.0.1",
 		"localhost",
+		".svc",
+		".cluster.local",
```

Comment:
This must be done in the installer as well

Reply:
cc @abhinavdahiya related to https://bugzilla.redhat.com/show_bug.cgi?id=1744532#c13

Reply:
attempted to fix this here https://github.com/openshift/installer/pull/2257